beispiel4.html

```diff
@@ -21,10 +21,7 @@
         <form name="eingabe_form" onsubmit="sendMsg()">
          
             <textarea id="txbx_eingabe" rows="3" cols="100" wrap="hard" ></textarea>
-            <!-- todo: should be in separate external script file -->
-            <script>
-                document.getElementById("txbx_eingabe").addEventListener("keypress", submitOnEnter);    
-            </script>    
+            <script type="text/javascript" src="scriptBeispiel4.js"></script>   
```

Comment:
the idea was to have 2 separate external javascript files:
- one in the meta sections with general js code which can be loaded before page is loaded
- second with js code that is related to DOM, so page must be loaded before